### ISSUE TITLE
Un-idling Jenkins via directly accessing jenkins.openshift.io results in HTTP 500 error

### ISSUE THREAD

**Issue by hferentschik** [fabric8-services/fabric8-jenkins-proxy#99]
> "code": "500",  "detail": "Could not login to Jenkins in manderse-jenkins namespace on behalf of the user manderse@redhat.com"

This happens when trying to directly access _jenkins.openshift.io_ when Jenkins is idled. The Jenkins pod gets scaled up, but then the actual login and forwarding does not work and results in the above error.

If one tries to access _jenkins.openshift.io_ after the pod is up, one gets successfully forwarded.

**Comment by xcoulon** [comment]
I confirm that I get the same error, with the following message:
````
{"Errors":[{"code":"500","detail":"Could not login to Jenkins in xcoulon-jenkins namespace on behalf of the user xcoulon@redhat.com"}]}
````

**Comment by hferentschik** [comment]
Thanks for the feedback. We are in the process of figuring out what's going on there.

**Comment by hferentschik** [comment]
Fixed via pull request #116
